### PR TITLE
test: Make testpulltask standalone

### PR DESCRIPTION
**Pull request by dperpeet**
For automated testing we want to use test pull requests as a standalone
task without communicating with GitHub.
As a side effect, this makes it easier to run the tests while developing.

### CODE REVIEW

**Comment by dperpeet** [comment]
@arilivigni this should make everything run on jenkins
it currently doesn't publish logs, but you could pipe them to a logfile

The command I used:
`./test/common/testpulltask.py -c verify/fedora-24 -n test-pull -v -o`
Before testing, a pull-request is rebased to `origin/master`, the offline option `-o` lets the script skip the fetch.

**Comment by mvollmer** [comment]
Can we tweak `github-task --head` to work standalone?  It looks like it is pretty close already.

**Comment by dperpeet** [comment]
> Can we tweak github-task --head to work standalone? It looks like it is pretty close already.

Instead of this?

**Comment by mvollmer** [comment]
> Instead of this?

Yes, so that instead of
```
./test/common/testpulltask.py -c verify/fedora-24 -n test-pull -v -o
```
we would run something like
```
./test/github-task --head verify/fedora-24 -n test-pull -v -o
```

We would need to add `-n` and `-o`, I guess.  Am I confused?

**Comment by dperpeet** [comment]
yeah, we could make that work

**Comment by mvollmer** [comment]
> yeah, we could make that work

I would prefer that since it doesn't add another (arguably obscure) entry point.

**Comment by arilivigni** [comment]
LGTM @dperpeet thanks!

**Comment by dperpeet** [comment]
It now works via something like `./test/github-task --head -n test-pull -v -o selenium/firefox`

File: test/common/testpulltask.py [hunk 1]
```diff
@@ -1,3 +1,22 @@
+#!/usr/bin/env python
```
Comment (mvollmer) [inline test/common/testpulltask.py]: This line is misleading now, no?  I remove this when merging.

**Pull request closed without merging**